app/assets/stylesheets/views/article.scss

```diff
@@ -33,7 +33,6 @@
         height: 100%;
         width: 100%;
         aspect-ratio: auto 650 / 275;
-        object-fit: scale-down;
```

Comment:
Unfortunately this means images that aren't wide enough will get squished (technical term 😅 ) - the original reason for the change was to make sure users could use different sizes of images and have them scaled to fit the container, rather than auto-enlarged to fill the whole width.

<img width="738" alt="Skewed cover image" src="https://user-images.githubusercontent.com/20773163/169855080-e28cbea6-0f2c-43f3-84f4-1b408c042396.png">

I'm not really sure at the minute why we get that peep of background color, given it should be occupying the full height 🤔